src/views/studio/VideoWorkspace/VideoWorkspace.style.ts

```diff
@@ -14,7 +14,7 @@ export const DrawerOverlay = styled.div`
   right: 0;
   bottom: 0;
   left: 0;
-  background-color: ${oldColors.transparentBlack[54]};
+  background-color: ${cVar('colorCoreNeutral500Darken')};
```

Comment:
Shouldn't this use a color token instead of `colorCore`?

Reply:
100%